ovis_robotiq_gripper/launch/robotiq_gripper.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
+  <arg name="robot_namespace" default="/ovis/gripper" />
+  <node name="ovis_robotiq_gripper" pkg="ovis_robotiq_gripper" type="ovis_gripper_control.py" ns="$(arg robot_namespace)" output="screen" args="/dev/ttyUSB0"></node>
```

Comment:
1. You don't have to add a closing tag if there's nothing in between.
2. I think we ought to have the `args="/dev/ttyUSB0"` as an args outside the node tag.

Reply:
Agreed, corrected as of now.